### PR TITLE
As a user of the Find and Replace feature, I want an easy way to see that replace is working

### PR DESCRIPTION
**Pull request by nambatee**
https://phabricator.wikimedia.org/T219666

https://github.com/wikimedia/wikipedia-ios-codemirror/pull/11

### CODE REVIEW

File: Wikipedia/assets/codemirror/resources/addon/search/search.js [hunk 1]
```diff
@@ -129,7 +129,23 @@
...
+      if (state.replacedMarkers) {
+        state.replacedMarkers.push(marker);
+      } else {
+        state.replacedMarkers = [marker];
+      }
+    }
+
+    function clearReplaced(state) {
+      if (state.replacedMarkers) {
+        state.replacedMarkers.forEach((marker) => { marker.clear() });
```
Comment (tonisevener) [inline Wikipedia/assets/codemirror/resources/addon/search/search.js]: Might want to set replacedMarkers to [] here too, otherwise it'll build up.

Comment (nambatee) [reply Wikipedia/assets/codemirror/resources/addon/search/search.js]: fixed

File: Wikipedia/assets/codemirror/resources/addon/search/search.js [hunk 2]
```diff
@@ -301,14 +320,15 @@
...
           cm.isReplacing = false;
         }
       }
-      var doReplace = function(match) {
+      var doReplace = function(cm, state) {
         cursor.replace(replaceText);
+        markReplacedText(cm, cursor);
         advance(false);
-        var forceIncrement = replaceText.includes(query) || replaceText.lowercase == query.lowercase;
+        var forceIncrement = replaceText.includes(query) || replaceText.toLowerCase() === query.toLowerCase();
```
Comment (nambatee) [inline Wikipedia/assets/codemirror/resources/addon/search/search.js]: Fixes wrong focus after single replace -
ezgif-4-953ae6f693aa

Comment (tonisevener) [reply Wikipedia/assets/codemirror/resources/addon/search/search.js]: ahh good catch, thanks

**Pull request merged**